lara-typescript/src/section-authoring/components/section-column.tsx

```diff
@@ -151,14 +161,14 @@ export const SectionColumn: React.FC<ISectionColumnProps> = ({
                     // TODO: Figure out why item.data property names are sometimes not converted
                     // to camel case, and fix it instead of using this itemData variable.
                     const itemData = snakeToCamelCaseKeys(item.data);
+                    if (isWrappingItem(itemData)) {
```

Comment:
Maybe add a comment here as to why we are exiting out early is it is a wrapping item?

Reply:
I agree a comment here is important. We are adding code that is depending on the componentLabels of specific plugins. So this means plugins are not generically handled. I think it is fine for now, so we can get this fixed, but you should comment about it.

Or if you could find out how to identify these wrapping items more generically that would be even better.